### DISCUSSION TITLE
Lets encrypt

### DISCUSSION THREAD

**Discussion by Ragemalik** [vilhelmprytz/pterodactyl-installer#211] [Q&A]
hey so i did not use lets encrypt in the setup but i wan to secure my pannel now what do i do?

**Comment by vilhelmprytz** [comment]
You would have to install it manually. The process of securing a website with Let’s Encrypt would be the same regardless of the web content served with Nginx. If you’re using Ubuntu, search for “nginx Let’s Encrypt Ubuntu” or something similar.

**Reply by Ragemalik** [reply]
Thank you for your reply.  I ak using my public ip how would I change it to my domain?

**Reply by Ragemalik** [reply]
Also when I ruined sudo reboot the panel dose not show anymore on the website do I need to do anything?
